targets/BootStrapMSBuild.proj

```diff
@@ -46,6 +46,23 @@
       <InstalledMicrosoftExtensions Include="$(MSBuildExtensionsPath)\Microsoft\**\*.*" />
 
       <InstalledNuGetFiles Include="$(MSBuildExtensionsPath)\Microsoft\NuGet\*" />
+      <!--<NuGetCommonExtensions Include="$(VsInstallRoot)\Common7\IDE\CommonExtensions\Microsoft\NuGet\**\*.*" />-->
+      <!-- TODO: Need NuGet.Build.Tasks.dll, would be not to not list out all of its dependencies! -->
+      <NuGetCommonExtensions Include="$(ProjectDir)packages\nuget.build.tasks\$(NuGetVersion)\lib\net45\**\*.*" />
```

Comment:
What was wrong with the initial way of getting them from under VS?

Potential problem with grabbing them from their packages is if they have a dependency closure that isn't covered by the enumerated packages.

Reply:
We need a new task from NuGet so we need to latest version. Getting it from VS won't work until that dependency has flowed through. Plus this is better anyway not to copy stuff from VS.